File: WebMiniMercadoLosPacos/src/main/java/com/minimercadolospacos/webminimercadolospacos/service/iface/ProductoService.java
package com.minimercadolospacos.webminimercadolospacos.service.iface;

import com.minimercadolospacos.webminimercadolospacos.models.entity.Producto;
import java.util.List;


public interface ProductoService {
    
    void create(Producto producto);
    
    List<Producto> getAll();
    
    void update(int id, Producto producto);
    
    void delete(int id);
}

File: projects react/webtienda-app/src/services/ProductoService.js
import {AxiosInstance} from "../config/axios-config"
class  ProductoService{
    get(){
        return AxiosInstance.get("producto");
   
    }

    delete(id){
        return AxiosInstance.delete(`producto/${id}`);
    }

    create(producto) {
        return AxiosInstance.post("producto", producto);
    }
}

export default new ProductoService();